packages/package_info/pubspec.yaml

```diff
@@ -2,10 +2,11 @@ name: package_info
 description: Flutter plugin for querying information about the application
   package, such as CFBundleVersion on iOS or versionCode on Android.
 homepage: https://github.com/flutter/plugins/tree/master/packages/package_info
-version: 2.0.0
+version: 2.0.1
 
 flutter:
   plugin:
+    implements: package_info
```

Comment:
package_info implements package_info? I thought implements was only for federated plugins.

Reply:
`default_package` is more appropriate here. The schema validator found `macos` platform, but no `implements:`, nor `default_package:` value.  Done